test/Core.Test/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommandTests.cs

```diff
@@ -235,7 +235,7 @@ await sutProvider.GetDependency<IEventService>().Received(0)
 
     [Theory]
     [BitAutoData]
-    public async Task DeleteManyUsersAsync_WithValidUsers_DeletesUsersAndLogsEvents(
+    public async Task DeleteManyUsersAsync_WhenFeatureFlagEnabled_WithValidUsers_DeletesUsersAndLogsEvents(
```

Comment:
Feature flag is not used
```suggestion
    public async Task DeleteManyUsersAsync_WithValidUsers_DeletesUsersAndLogsEvents(
```